From 4413c8788ad2daed62277e836c17998942e96436 Mon Sep 17 00:00:00 2001
From: Alan Ott <alan@signal11.us>
Date: Sat, 3 Jan 2015 14:47:40 -0500
Subject: [PATCH 29/29] audio-cape: Add comments

Describe why the pinmux settings are disabled.
Tell that one of the pinmux settings has no "audio" mode.

Signed-off-by: Alan Ott <alan@signal11.us>
---
 arch/arm/boot/dts/am335x-bone-audio-revb.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/am335x-bone-audio-revb.dtsi b/arch/arm/boot/dts/am335x-bone-audio-revb.dtsi
index ef09972..98e49e8 100644
--- a/arch/arm/boot/dts/am335x-bone-audio-revb.dtsi
+++ b/arch/arm/boot/dts/am335x-bone-audio-revb.dtsi
@@ -13,6 +13,9 @@
 	P9_20_pinmux {
 		mode = "i2c";
 	};
+
+	/* The following are disabled because the audio mode is set
+	   by <&mcasp0_pins_audio_revb>. */
 	P9_25_pinmux {
 		/* mcasp0_ahclkx, OUTPUT | MODE0, Codec MCLK */
 		status = "disabled";
@@ -28,6 +31,8 @@
 	P9_30_pinmux {
 		/* mcasp0_axr0, INPUT | MODE0, Codec DOUT */
 		status = "disabled";
+		/* TODO Add support in am335x-bone-common-pinmux.dtsi
+		 * for an "audio" mode. */
 	};
 	P9_31_pinmux {
 		/* mcasp0_aclkx, INPUT | MODE0, Codec BCLK */
-- 
2.1.4

